recipe/meta.yaml

```diff
@@ -213,17 +214,17 @@ outputs:
         - {{ pin_subpackage('mkl', exact=True) }}
         - {{ pin_subpackage('mkl-include', exact=True) }}
         - blas * mkl
+    test:
+      commands:
+        - IF NOT EXIST %PREFIX%\Library\lib\mkl*dll.lib EXIT /B 1  # [win]
+        # Version 2023.1.0-intel_46342 for linux-64 and 2023.1.0-intel_43558 for osx-64 do not have a lib folder,
```

Comment:
I think you want to say here that no `.dll` or `.lib` files are included here, since the `lib` folder does exist on all platforms (but it only has `pkgconfig` files for `osx-64` and `linux-64`).

Reply:
Fixed